docs/quickstart/regression.md

```diff
@@ -0,0 +1,122 @@
+---
+jupytext:
+  text_representation:
+    extension: .md
+    format_name: myst
+    format_version: 0.13
+    jupytext_version: 1.14.1
+kernelspec:
+  display_name: Python 3 (ipykernel)
+  language: python
+  name: python3
+---
+
+# Regression
+
++++
+
+Regression analysis is a set of statistical methodologies for determining the relationship between a dependent (or outcome) variable and one or more independent variables (also known as predictor variables). The most common algorithm used is Linear Regression. 
+
+Linear Regression is a supervised learning technique that attempts to fit the best line that reduces the discrepancies between the predicted and the actual target value. This best fit line is arrived at by aiming to reduce the MSE (Mean Square Error).
+
++++
+
+Let's import the required libraries and read the dataset.
+
+```{code-cell} ipython3
+import urllib.request
+import pandas as pd 
+import seaborn as sns
+import warnings
+warnings.filterwarnings('ignore')
+from sklearn.preprocessing import LabelEncoder
+from sklearn.linear_model import LinearRegression
+from sklearn.model_selection import train_test_split
+from sklearn.metrics import r2_score
+from sklearn_evaluation import plot
+
+# download dataset
+urllib.request.urlretrieve('https://raw.githubusercontent.com/ploomber/sklearn-evaluation/master/docs-assets/regression/insurance.csv', filename='insurance.csv')
+
+data = pd.read_csv('insurance.csv')
```

Comment:
where is this data coming from? we need to cite the source and ensure the license allows us to use it. furthermore, it's better to download it from the original source instead of copying it and hosting it ourselves

Reply:
Added reference. License: https://opendatacommons.org/licenses/dbcl/1-0/